fbc/v4.22/Dockerfile

```
@@ -0,0 +1,22 @@
+# The builder image is expected to contain
+# /bin/opm (with serve subcommand)
+FROM brew.registry.redhat.io/rh-osbs/openshift-ose-operator-registry-rhel9:v4.22 as builder
+
+# Copy FBC root into image at /configs and pre-populate serve cache
+ADD catalog/ /configs
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)